notebooks/001-hello-world/001-hello-world.ipynb

```diff
@@ -2,131 +2,120 @@
    "source": [
     "## Load the network"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "statewide-growth",
-   "metadata": {},
-   "outputs": [],
    "source": [
     "ie = IECore()\n",
     "net = ie.read_network(\n",
-    "    model=\"v3-small_224_1.0_float.xml\", weights=\"v3-small_224_1.0_float.bin\"\n",
+    "    model=\"model/v3-small_224_1.0_float.xml\", weights=\"model/v3-small_224_1.0_float.bin\"\n",
     ")\n",
     "exec_net = ie.load_network(net, \"CPU\")\n",
     "\n",
     "input_key = list(exec_net.input_info)[0]\n",
     "output_key = list(exec_net.outputs.keys())[0]"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "id": "convertible-literacy",
-   "metadata": {},
    "source": [
     "## Load an Image"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "simplified-accommodation",
-   "metadata": {},
-   "outputs": [],
    "source": [
     "# The MobileNet network expects images in RGB format\n",
-    "image = cv2.cvtColor(cv2.imread(\"coco.jpg\"), cv2.COLOR_BGR2RGB)\n",
+    "image = cv2.cvtColor(cv2.imread(\"data/coco.jpg\"), cv2.COLOR_BGR2RGB)\n",
     "input_image = cv2.resize(image, (224, 224))  # resize to MobileNet image shape\n",
     "input_image = np.expand_dims(\n",
     "    input_image.transpose(2, 0, 1), 0\n",
     ")  # reshape to network input shape\n",
     "plt.imshow(image)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "id": "warming-platinum",
-   "metadata": {},
    "source": [
     "## Do Inference"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "italic-comment",
-   "metadata": {},
-   "outputs": [],
    "source": [
     "result = exec_net.infer(inputs={input_key: input_image})[output_key]\n",
     "result_index = np.argmax(result)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "complicated-command",
-   "metadata": {},
-   "outputs": [],
    "source": [
     "# Convert the inference result to a class name.\n",
-    "imagenet_classes = json.loads(open(\"imagenet_class_index.json\").read())\n",
+    "imagenet_classes = json.loads(open(\"utils/imagenet_class_index.json\").read())\n",
     "# The model description states that for this model, class 0 is background,\n",
     "# so we add 1 to the network output to get the class name\n",
     "imagenet_classes = {\n",
     "    int(key) + 1: value for key, value in imagenet_classes.items()\n",
     "}\n",
     "imagenet_classes[result_index]"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "openvino_env",
-   "language": "python",
-   "name": "openvino_env"
+   "name": "python3",
+   "display_name": "Python 3.9.6 64-bit (windows store)"
```

Comment:
@Debskij was this a mistake? This version of Python is not supported

Reply:
Fixed